### PR TITLE
Add auth.user to profiles upon signup

### PR DESCRIPTION
**Pull request by evanfordebarden**
When a user signs up, we now have a flow that makes sure a profile is created for them in the profiles table.

1. user enters signup info
2. they receive confirmation email
3. upon confirmation, they're directed to /welcome, where a welcome modal is displayed
4. when /welcome mounts, we create a profile for the user.
5. in the background, we load all moments.
6. if they click close, they're redirected to /moments.
7. if they click anything else, the welcome modal closes, showing them all moments (but still @ /welcome)

changed some things around in App.js as well so make sure to review what's going on there.

### CODE REVIEW

**Review by danielyj98 — COMMENTED** [review]
Looks great! Would it be possible (or even worthwhile) that when the user clicks outside the modal that it _**would**_ redirect them to /moments? Or was that intentional/the only way to go about it? Just wanted to get clarification on that before merging, but other than that, looks freaking incredible 🤘

**Comment by evanfordebarden** [comment]
> 

@danielyj98 that would be ideal, but i couldn't figure out how to do it with the chakra component. i tried disabling the default action (closing the modal) and adding an onClick to navigate to /messages, but that didn't work. it keeps the modal open but doesn't re-route. i'm going to add a ticket to try and sort that out, but i think for mvp this works. down to review it tomorrow and make a final call.

**Comment by danielyj98** [comment]
Thought so lol, just wanted to make sure- it's totally functional so I'll just merge

**Review by danielyj98 — APPROVED** [review]

**Pull request merged**
